gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-container/pom.xml

```diff
@@ -145,6 +145,12 @@
             <artifactId>jul-to-slf4j</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Is that dependency addition related to this feature ?

Reply:
nop, I add some issues running 3.5.x in ee mode and I unfortunately commit :)